BeatSaberMarkupLanguage/TypeHandlers/RectTransformHandler.cs

```diff
@@ -24,6 +24,7 @@ public class RectTransformHandler : TypeHandler<RectTransform>
             { "hoverHint", new[] { "hover-hint" } },
             { "hoverHintKey", new[] { "hover-hint-key" } },
             { "active", new[] { "active" } },
+            { "localScale", new[] { "local-scale", "scale", "size" } },
```

Comment:
I'm not a huge fan of the `size` alias. It think it could lead to confusion considering `sizeDelta` is a property on `RectTransform`. I'd rather stick with just the first two.